src/lib/constants.ts

```diff
@@ -1,3 +1,6 @@
 export const Bech32AddressLength = 63;
 export const EVMAddressLength = 42;
 export const NativeTokenIDLength = 38;
+
+export const BASE_TOKEN_DECIMALS = 18;
```

Comment:
I would rename this to clarify is the L2 base token

Reply:
We could also make use of these constants in other places like
- https://github.com/trungtt198x/evm-toolkit/blob/ee02adfc169e114d3426baf7d53ddf0bb87e00c1/src/lib/withdraw/actions/add-network-to-metamask.ts#L21-L22
- https://github.com/trungtt198x/evm-toolkit/blob/ee02adfc169e114d3426baf7d53ddf0bb87e00c1/src/lib/withdraw/actions/add-token-to-metamask.ts#L20
- https://github.com/trungtt198x/evm-toolkit/blob/ee02adfc169e114d3426baf7d53ddf0bb87e00c1/src/routes/sections/wrap-section.svelte#L201
wdyt?

Reply:
Agree